src/main/java/com/dream11/aerospike/client/AerospikeClientImpl.java

```diff
@@ -91,14 +128,14 @@ public void close() {
    * The policy specifies the transaction timeout, record expiration and how the transaction is
    * handled when the record already exists.
    *
-   * @param writePolicy	 write policy details
-   * @param key					unique record identifier
-   * @param bins				array of bin name/value pairs
+   * @param writePolicy write policy details
```

Comment:
I think we should keep the documentation about the method and its parameters in the interface itself. In the implementation, we should only put details of implementation.